### ISSUE TITLE
Connection lost every hour (livetrack)

### ISSUE THREAD

**Issue by SPalette** [tinuzz/wp-plugin-trackserver#36]
Hallo Mr. Grendelman,

i tried do bring location data via curl to trackservers osmand-interface. Now livetracking works fine but every track recording ends on the last waypoint in a hour. For example, a track recording works up to 15:59:02 assigned to track #1, then the next waypoint 16:00:02 will be assigned to track #2. While livetracking an alertbox appears (Track missing from server response. Please reload the page). Unfortunately, only the waypoints of on (#2) track are displayed after page reloading....

All other trackserver-functions are working fine...

best regards
Stephan

**Comment by tinuzz** [comment]
Hi Stephan,

Please look at the "OsmAnd trackname format" in the options page. It likely contains '%H' or something similar, which means that Trackserver generates a new track every hour.

Regards,
Martijn.

**Comment by SPalette** [comment]
Yeah, I got it yesterday in the evening while a extensively code review. :-)
In this context I gleaned, that strftotime() now is deprecated (PHP 8.1). Anyway, it works...
What not really works is to save a new password. The request crashes. (PHP 8.2, Apache 5, Win)
I solved this problem by deleting the whole "action" - parameter include the inline-php-statement in class-trackserver-para file.php (var $url is not set in that context).

Thank you for your work, and
best regards
Stephan
